js/renovation/ui/form/form.tsx

```diff
@@ -31,15 +38,19 @@ export const viewFunction = (viewModel: Form): JSX.Element => {
         bounceEnabled={false}
         // eslint-disable-next-line react/jsx-props-no-spreading
         // TODO: {...restAttributes} waits for the https://trello.com/c/er8aTcsZ/2711-renovation-react-some-events-from-restattributes-may-have-the-same-type-as-react-exists-events
-      />
+      >
+        {rootLayoutManager}
+      </Scrollable>
     )
     : (
       <Widget
         aria={aria}
         classes={cssClasses}
         // eslint-disable-next-line react/jsx-props-no-spreading
         {...restAttributes}
-      />
+      >
+        {rootLayoutManager}
```

Comment:
Whether we should replace Widget with rootLayoutManager ?

In old code it was:
    _getContent: function() {
        return this.option('scrollingEnabled') ? this._scrollable.$content() : this.$element();
    },

Reply:
No, no. When scrolling is disabled, then layout manager is wrapped by the div with aria attributes and "dx-widget form" css classes
![image](https://user-images.githubusercontent.com/57402891/116647057-4f86a980-a982-11eb-97d0-976c9f86059a.png)


In the old code, when scrolling is enabled, then scrollable is applied to the main div. So main div is combination of the 2 widgets: form and scrollable - https://github.com/DevExpress/DevExtreme/blob/37ab2c57d466b499a460557cd1bed95fc9ef29d9/js/ui/form/ui.form.js#L357